npc/scripts_monsters.conf

```diff
@@ -2,7 +2,6 @@
 // -                      Monster  Scripts                      -
 // --------------------------------------------------------------
 
-npc: npc/mobs/citycleaners.txt
```

Comment:
citycleaners path for pre-re is missing in this commit

Reply:
Pre-RE was added to, not removed in this commit. 
`npc: npc/pre-re/mobs/citycleaners.txt` is defined in npc/pre-re/scripts_monsters.conf